src/vs/workbench/services/workspaces/common/workspaceTrust.ts

```diff
@@ -86,14 +85,17 @@ export class WorkspaceTrustManagementService extends Disposable implements IWork
 	private readonly _onDidChangeTrustedFolders = this._register(new Emitter<void>());
 	readonly onDidChangeTrustedFolders = this._onDidChangeTrustedFolders.event;
 
-	private _trustStateInfo: IWorkspaceTrustInfo;
 	private _canonicalWorkspace: IWorkspace;
+	private _canonicalOpenFiles: URI[] = [];
+	private _canonicalUrisResolved: boolean;
 
-	protected readonly _trustState: WorkspaceTrustState;
+	private _isTrusted: boolean;
+	private _trustStateInfo: IWorkspaceTrustInfo;
+	private readonly _trustState: WorkspaceTrustState;
```

Comment:
suggest to rename this to `_storedTrustState` to amplify the distinction between this value and the `_isTrusted` property.